content/maxwell1_fundamentals/harmonic_planewaves_homogeneous/skindepth.rst

```diff
@@ -18,7 +18,7 @@ Attenuation defines the rate of amplitude loss an EM wave experiences at it prop
 .. _harmonic_planewaves_homogeneous_attenuation_formula:
 
 .. math::
-	A(z) = A_0 e^{\beta z}
+	A(z) = A_0 e^{-\beta z}
```

Comment:
must be minus in order to attenuate the amplitude.

Reply:
If it is positive, the "z" values must be negative. I think we can clarify this in some sentences. For example "z subsurface is between zero and -infinity". That is because, in several books, the "z" is positive, thus, the beta value in the amplitude equation must be with a minus.